bin/prepare_environment.sh

```diff
@@ -16,41 +18,44 @@ else
     exit 1
 fi
 
+echo ""
+
 ## Clean up previous installations
-directories=("status-backend" "status-frontend" "node-red-status" "mysql" "collector-events")
+directories=("status-backend" "status-frontend" "node-red-status" "collector-events" ".env")
 
 for dir in "${directories[@]}"; do
-    if [ -d "../$dir" ]; then
+    if [ -d "$dir" ]; then
         echo "Deleting $dir..."
-        rm -rf "../$dir"
+        rm -rf "$dir"
     fi
 done
 
+echo ""
 
 ## Clone repositories
 
 echo "_______________________CLONING REPOSITORIES_______________________"
 
-git clone -b feat/9-Manage_AI_requests https://github.com/statuscompliance/status-backend.git ../status-backend
+git clone -b feat/9-Manage_AI_requests https://github.com/statuscompliance/status-backend.git status-backend
 
-git clone -b feature/24-Bluejay_integration https://github.com/statuscompliance/status-frontend.git ../status-frontend
+git clone -b feature/24-Bluejay_integration https://github.com/statuscompliance/status-frontend.git status-frontend
```

Comment:
To deploy status-frontend with nginx, you need to use the last frontend version (located in `feature/22-Create_initial_chatbot` branch).

Reply:
Updated 👍